src/routes/web.php

```diff
@@ -3,14 +3,32 @@
 use Illuminate\Support\Facades\Route;
 use App\Http\Controllers\PostController;
 use App\Http\Controllers\UserController;
-use App\Http\Controllers\LoginController;
-use App\Http\Controllers\RegisterController;
+use App\Http\Controllers\Auth\LoginController;
+use App\Http\Controllers\Auth\RegisterController;
+use App\Http\Controllers\Auth\VerificationController;
+use App\Http\Controllers\Auth\ResetPasswordController;
+use App\Http\Controllers\Auth\ForgotPasswordController;
 
-Route::get('/', [LoginController::class, 'index'])->name('view.login');
-Route::post('/index', [LoginController::class, 'auth'])->name('login.auth');
+Route::middleware(['auth', 'verified'])->group(function () {
+    Route::get('/profile/show', [UserController::class, 'show'])->name('profile.show');
+    Route::post('/logout', [UserController::class, 'logout'])->name('user.logout');
+    Route::get('/home', [UserController::class, 'home'])->name('user.home');
+    Route::post('/posts', [PostController::class, 'store'])->name('post.store');
+});
+
+Route::get('/', [LoginController::class, 'index'])->name('login');
+Route::post('/login', [LoginController::class, 'auth'])->name('login.auth');
 Route::get('/register', [RegisterController::class, 'create'])->name('register.create');
 Route::post('/register', [RegisterController::class, 'store'])->name('register.store');
-Route::get('/profile/show', [UserController::class, 'show'])->name('profile.show');
-Route::post('/logout', [UserController::class, 'logout'])->name('user.logout');
-Route::get('/home', [UserController::class, 'home'])->name('user.home');
-Route::post('/posts', [PostController::class, 'store'])->name('post.store');
+
+
+// Password Reset Routes...
+Route::get('password/reset', [ForgotPasswordController::class, 'showLinkRequestForm'])->name('password.request');
+Route::post('password/email', [ForgotPasswordController::class, 'sendResetLinkEmail'])->name('password.email');
+Route::get('password/reset/{token}', [ResetPasswordController::class, 'showResetForm'])->name('password.reset');
+Route::post('password/reset', [ResetPasswordController::class, 'reset'])->name('password.update');
+
+// Email Verification Routes...
+Route::get('email/verify', [VerificationController::class, 'show'])->name('verification.notice');
+Route::get('email/verify/{id}/{hash}', [VerificationController::class, 'verify'])->name('verification.verify');
+Route::get('email/resend',  [VerificationController::class, 'resend'])->name('verification.resend');
```

Comment:
Why still no middleware? Research about Guest Middleware in laravel!

Reply:
I already resolved this Sir according to your comment, thank you!